tests/unit_test/chat/chat_test.py

```
@@ -936,7 +936,7 @@ async def test_mongotracker_save(self):
         assert len(data) == len(events)
         assert data[0]['tag'] == 'tracker_store'
         assert data[0]['type'] == 'bot'
-        data = list(store.client.get_database(config['db']).get_collection(bot).find({'type': 'flattened'}))
+        data = list(store.client.get_database(f"{config['db']}").get_collection(f"flattened_conversations").find({'type': 'flattened'}))
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)